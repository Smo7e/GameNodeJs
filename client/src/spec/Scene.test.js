import React from 'react';
import { render } from '@testing-library/react';
import Scene from '../modules/Game/Scene';

describe('Scene', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Scene />);
    expect(asFragment()).toMatchSnapshot();
  });
});
