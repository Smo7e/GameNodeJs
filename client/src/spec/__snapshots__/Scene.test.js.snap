// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scene renders correctly 1`] = `<DocumentFragment />`;
